### ISSUE TITLE
Messages never shows for exceptions

### ISSUE THREAD

**Issue by IgrekYY** [zendesk/sunshine-conversations-ruby#20]
Hi!

There is no error message when exception from SmoochApi raises:

```
ETHON: performed EASY effective_url=https://api.smooch.io/v1.1/whatsapp/deployments/12312312/code/confirm response_code=400 return_code=ok total_time=0.181729
SmoochApi::ApiError: 
```

Always returns empty message. There are may be some extra setup?

**Comment by haziqhafizuddin** [comment]
Am getting this error as well, is there update on this?

**Comment by apocm** [comment]
When displaying the error as a string (eg. `puts error`), there was no message to be displayed; this has been fixed in version 5.34.1. Otherwise, to view the data you could also inspect your error object, eg. `error.inspect`
